tools/PowerShell/Microsoft.WinGet.Client/src/Library/Uninstall-WinGetPackage.ps1

```diff
@@ -0,0 +1,89 @@
+Function Uninstall-WinGetPackage{
```

Comment:
Is this file supposed to contain functionality for uninstalling? If it is, there isn't any function there